app/models/User.coffee

```diff
@@ -180,6 +180,19 @@ module.exports = class User extends CocoModel
 
   isOnPremiumServer: ->
     me.get('country') in ['china', 'brazil']
+    
+  spy: (target, options={}) ->
+    target = target.id or target
```

Comment:
When would target.id exist?  It seems like this is always name or email.  If it can really be an Id of some sort, that's at odds with the post data called nameOrEmailLower.